src/models/__tests__/profile.test.ts

```diff
@@ -117,6 +118,27 @@ describe("findLastVersionByModelId", () => {
   });
 });
 
+describe("Profile codec", () => {
+  it("should consider all possible ServicesPreferencesMode values", async () => {
+    // This is a safe-guard to programmatically ensure all possible values of ServicesPreferencesModeEnum are considered
+
+    expect.assertions(0);
```

Comment:
Do we need this?

Reply:
Actually no, I was sure jest would have complained with a tests without assertions. 

Removed in https://github.com/pagopa/io-functions-commons/pull/185/commits/9dc766413db7cb780209af74a1c641286335877a

Thanks